### PR TITLE
Update NPM dependencies

### PR DESCRIPTION
**Pull request by frankrousseau**

### CODE REVIEW

**Comment by nono** [comment]
@frankrousseau what is the status of this PR?

**Comment by frankrousseau** [comment]
I'm waiting for someone to review and test it.

File: package.json [hunk 1]
```diff
@@ -12,29 +12,29 @@
...
-    "googleapis": "2.0.2",
-    "imagemagick-stream": "^2.0.0",
-    "jade": "1.9.1",
-    "lodash": "^3.8.0",
-    "moment-timezone": "^0.3.1",
-    "node-polyglot": "^0.4.1",
-    "printit": "0.1.15",
+    "gm": "1.22.0",
+    "googleapis": "4.0.0",
+    "imagemagick-stream": "4.0.1",
```
Comment (nono) [inline package.json:22]: This version is not compatible with node 0.10, see https://github.com/cozy-labs/konnectors/commit/fc91e5318312d0b44095b476769a127aaf808756

**Comment by nono** [comment]
@poupotte can you test it? Or, do you know someone else who can do that? I'm not very confident to update a lot of nodejs dependencies on a project with no travis build :wink:

**Comment by frankrousseau** [comment]
Let's close it there is no hurry there.

**Pull request closed without merging**